compiler/rustc_lint/src/builtin.rs

```diff
@@ -3157,3 +3157,39 @@ impl<'tcx> LateLintPass<'tcx> for NamedAsmLabels {
         }
     }
 }
+
+use rustc_session::lint::builtin::UNEXPECTED_CFGS;
```

Comment:
Should I create new lint for the (new ?) warning or re-using `UNEXPECTED_CFGS` is fine ?

Btw, this re-using is the reason why I needed to update `compiler/rustc_lint_defs/src/builtin.rs` and `src/librustdoc/core.rs`.

Reply:
I think it's better to reuse `unexpected_cfgs` here.